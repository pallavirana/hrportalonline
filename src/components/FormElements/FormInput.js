import React from 'react';
import withStyles from 'react-jss';
import styles from './FormElements.styles';

export const FormInput = ({classes, type, name, placeholder}) => <div className={classes.FormInputWrap}>
    <input type={type} name={name} placeholder={placeholder} className={classes.FormInput} />
</div>;

export default withStyles(styles)(FormInput);