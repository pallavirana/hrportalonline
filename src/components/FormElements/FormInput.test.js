import React from 'react';
import { shallow } from 'enzyme';
import { FormInput } from "./FormInput";


describe('FormInput component', () => {
    it('shallow renders without crashing', () => {
      const wrapper = shallow(<FormInput classes={{}} />);
      expect(wrapper).toMatchSnapshot();
    });
  });