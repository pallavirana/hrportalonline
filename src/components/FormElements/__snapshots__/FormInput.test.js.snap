// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput component shallow renders without crashing 1`] = `ShallowWrapper {}`;
